Jenkinsfile

```diff
@@ -16,7 +16,7 @@
 // Then a second PR submitted to comment out the @Library line, and when it
 // is landed, both PR branches can be deleted.
 
-// @Library(value="pipeline-lib@my_pr_branch") _
+@Library(value="pipeline-lib@corci-918e") _
```

Comment:
is the branch here going to cause an issue?

Reply:
We have to remove it after the merge.  It was added by a github action as an experiment with having PRs auto-update it.

It only affects the local tests for the pipeline-lib